server/middleware/reactApplication/index.js

```diff
@@ -1,36 +1,43 @@
-import React from 'react';
-import Helmet from 'react-helmet';
-import { renderToString, renderToStaticMarkup } from 'react-dom/server';
-import { StaticRouter } from 'react-router-dom';
-import { AsyncComponentProvider, createAsyncContext } from 'react-async-component';
-import asyncBootstrapper from 'react-async-bootstrapper';
+import React from "react";
```

Comment:
Why do you use double quotes? Have you linted your code?

Reply:
Looks like `prettier` didn't run properly?  FYI all I have updated prettier.  We no longer use `prettier-eslint`.  Just use native `prettier` and `eslint` separately.  I installed the `eslint-config-prettier` plugin which disables all the style rules so we can still use the airbnb config.